src/sparseml/keras/optim/manager.py

```diff
@@ -22,6 +22,12 @@
 
 import tensorflow as tf
 
+
+try:
+    import keras
+except ModuleNotFoundError:
```

Comment:
This catch needs to be more generic like with Exception. We also should have some logging built-in around this at least at the debug level